pkg/models/measurement.go

```diff
@@ -0,0 +1,50 @@
+package models
+
+import "github.com/grafana/grafana-plugin-sdk-go/data"
+
+// NOTE:
+// this likely shoud go in the Plugin SDK since it will be useful from plugins
+
+// Measurement is a single measurement value
+type Measurement struct {
+	// the measurement name
+	Name string `json:"name,omitempty"`
+
+	// units are usually ms, but depend on the channel
+	Time int64 `json:"time,omitempty"`
+
+	// typically number or string
+	Values map[string]interface{} `json:"values,omitempty"`
+
+	// optional list of field configs
+	Config map[string]data.FieldConfig `json:"config,omitempty"`
+
+	// labels are applied to all values
```

Comment:
What is the key and value in this map?
```suggestion
	// Labels are applied to all values.
```